webapp/i18n/en.json

```diff
@@ -1267,6 +1267,8 @@
   "edit_post.edit": "Edit {title}",
   "edit_post.editPost": "Edit the post...",
   "edit_post.save": "Save",
+  "edit_channel_header.editHeader": "Edit the Channel Header...",
+  "edit_channel_header.previewHeader": "Edit Header",
```

Comment:
Argh, sorry -- this should be in sentence case: `Edit header`

My fault, I entered the text incorrectly.